crates/l2/contracts/src/l1/OnChainProposer.sol

```diff
@@ -99,6 +131,15 @@ contract OnChainProposer is IOnChainProposer, ReentrancyGuard {
             "OnChainProposer: block already verified"
         );
 
+        if (R0VERIFIER != address(0xAA)) {
```

Comment:
We should add a constant with the value `address(0xAA)` documenting what it represents.